x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/product_selector/enterprise_search_product_card.tsx

```diff
@@ -19,23 +19,45 @@ import { ProductCard } from '../product_card';
 import { AppSearchProductCard } from './app_search_product_card';
 import { WorkplaceSearchProductCard } from './workplace_search_product_card';
 
-export const EnterpriseSearchProductCard = () => (
-  <ProductCard
-    description={i18n.translate('xpack.enterpriseSearch.entSearch.productCardDescription', {
-      defaultMessage:
-        'Standalone applications tailored to simpler, user-friendly and business-focused search experiences.',
-    })}
-    emptyCta
-    cta={i18n.translate('xpack.enterpriseSearch.enterpriseSearchCard.cta', {
-      defaultMessage: 'Learn more',
-    })}
-    url={docLinks.start}
-    icon="logoEnterpriseSearch"
-    name={ENTERPRISE_SEARCH_PRODUCT_NAME}
-    productId={ENTERPRISE_SEARCH_CONTENT_PLUGIN.ID}
-    rightPanelItems={[
-      <AppSearchProductCard hasBorder={false} hasShadow={false} />,
-      <WorkplaceSearchProductCard hasBorder={false} hasShadow={false} />,
-    ]}
-  />
-);
+export interface EnterpriseSearchProductCardProps {
+  hasAppSearchAccess: boolean;
+  hasWorkplaceSearchAccess: boolean;
+  isWorkplaceSearchAdmin: boolean;
+}
+
+export const EnterpriseSearchProductCard = ({
+  hasAppSearchAccess,
+  hasWorkplaceSearchAccess,
+  isWorkplaceSearchAdmin,
+}: EnterpriseSearchProductCardProps) => {
+  const rightPanelItems: React.ReactNode[] = [];
+  if (hasAppSearchAccess) {
```

Comment:
Is there ever a use case where this will be empty (user doesn't have app search _or_ workplace search access)?

Reply:
yes, when `ent-search` node is not running.

Reply:
Updated to handle this case and fix existing test, added tests for all the update permutations.